src/Fixer/Operator/StandardizeIncrementFixer.php

```diff
@@ -135,8 +135,14 @@ private function findStart(Tokens $tokens, $index)
             $index = $tokens->getPrevMeaningfulToken($index);
         }
 
-        if ($tokens[$tokens->getPrevMeaningfulToken($index)]->isGivenKind(T_OBJECT_OPERATOR)) {
-            return $this->findStart($tokens, $tokens->getPrevMeaningfulToken($index));
+        $prevIndex = $tokens->getPrevMeaningfulToken($index);
+
+        if ($tokens[$prevIndex]->isGivenKind(T_DOUBLE_COLON)) {
+            return $tokens->getTokenNotOfKindSibling($prevIndex, -1, [[T_NS_SEPARATOR], [T_STATIC], [T_STRING], [T_VARIABLE]]) + 1;
```

Comment:
Realizing that `$a->$b->$c += 1;` is handled recursively (below),
do we want to handle `$a::$b::$c += 1;` too? and mixed `$a->$b::$c += 1;` / `$a::$b->$c += 1;`? (maybe not...)

Reply:
Good catch, even too good, I think we need some more sophisticated check, see the test cases I've added.

Reply:
I see you managed to sort it out, good job! 👍

Reply:
@guilliamxavier did you take a look at the rest of PR to leave a review?

Reply:
Yes I did look, I just didn't think to leave a review 😅

Reply:
Thanks